auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -57,9 +111,9 @@ class CheckPullRequest extends RequestHandler {
     final GithubService gitHub = await config.createGithubService();
     final _AutoMergeQueryResult queryResult = await _parseQueryData(pullRequest, gitHub);
     if (await shouldMergePullRequest(queryResult, slug, gitHub)) {
-      log.info('Merge the pull request: ${queryResult.number}');
-      // TODO(Kristin): Merge this PR. https://github.com/flutter/flutter/issues/100088
-      return Response.ok('Merge the pull request ${queryResult.number} in ${slug.fullName} repository.');
+      log.info('Should merge the pull request: ${queryResult.number}');
+      shouldMergeSet.add(pullRequest);
```

Comment:
A map is more suitable her to make the logic simpler: repo->set of mergeable PRs

Reply:
Gotcha Keyong, I will create a map here to simply the following logics, thank you.